test-codegen-fix.js

```
@@ -0,0 +1,18 @@
+// Test file with intentional errors for Codegen auto-fix verification
+// Codegen should automatically fix these issues when CI fails
+
+const testFunction = () => {
+    console.log("Missing semicolon")  // Missing semicolon
+
+    const unusedVariable = "This is never used";
+
+    // Missing closing brace
+    if (true) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)